javascript/javascript1/week1/js-week1.js

```diff
@@ -0,0 +1,82 @@
+// Smart-ease
+// Age-ify (A future age calculator)
+const yearOfBirth = 1975;
+const yearFuture = 2045;
+
+let age = yearFuture - yearOfBirth;
+// console.log(age);
+
+console.log("You will be " + age + " " + "years old in " + yearFuture + ".");
```

Comment:
Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals